en/docs/reference/connectors/jira-connector/jira-connector-config.md

```diff
@@ -12,28 +13,28 @@ The following operations allow you to work with the Jira Connector. Click an ope
         </tr>
         <tr>
             <td>username</td>
-            <td>The username of the user.</td>
+            <td>The username of the Jira account.</td>
             <td>Yes</td>
         </tr>
         <tr>
             <td>password</td>
-            <td>The password of the user.</td>
+            <td>The password associated with the Jira account.</td>
```

Comment:
In the [description](https://github.com/wso2/docs-mi/pull/1630/files#diff-aa227a2f65993c5ccea962892697d3c4c292064ba8314d53e7aef1798b934238R5), it is mentioned we can use an `API token`, if so shall we update the description of the `password` field as well?